tasktiger/logging.py

```diff
@@ -0,0 +1,14 @@
+from _internal import g
+
+
+def tasktiger_processor(logger, method_name, event_dict):
+    """
+    TaskTiger structlog processor.
+
+    Inject the current task id for non-batch tasks.
+    """
+
+    if g['current_tasks'] is not None and not g['current_task_is_batch']:
+        event_dict['task_id'] = g['current_tasks'][0].id
```

Comment:
Is it possible for `g['current_tasks']` to be `[]` here? Or is that value either a `None` or a non-empty list?

Reply:
I kind of based that on [this code](https://github.com/closeio/tasktiger/blob/fda9d01e9eb5e31508a1420a3d11367f85c4c2f2/tasktiger/__init__.py#L208) which looks like we already assume `g['current_tasks']` won't be `[]`.